contracts/FlowIDTableStaking.cdc

```diff
@@ -1069,15 +1071,17 @@ pub contract FlowIDTableStaking {
             
             let allNodeIDs = FlowIDTableStaking.getNodeIDs()
 
+            let stakedNodeIDs: {String: Bool} = FlowIDTableStaking.getParticipantNodeList()!
+
             for nodeID in allNodeIDs {
                 let nodeRecord = FlowIDTableStaking.borrowNodeRecord(nodeID)
 
-                FlowIDTableStaking.totalTokensStakedByNodeType[nodeRecord.role] = FlowIDTableStaking.totalTokensStakedByNodeType[nodeRecord.role]! + nodeRecord.tokensCommitted.balance
-
                 // mark the committed tokens as staked
                 if nodeRecord.tokensCommitted.balance > 0.0 {
+                    FlowIDTableStaking.totalTokensStakedByNodeType[nodeRecord.role] = FlowIDTableStaking.totalTokensStakedByNodeType[nodeRecord.role]! + nodeRecord.tokensCommitted.balance
                     emit TokensStaked(nodeID: nodeRecord.id, amount: nodeRecord.tokensCommitted.balance)
                     nodeRecord.tokensStaked.deposit(from: <-nodeRecord.tokensCommitted.withdraw(amount: nodeRecord.tokensCommitted.balance))
+                    stakedNodeIDs[nodeRecord.id] = true
```

Comment:
Shouldn't we check for the minimum stake here?

Reply:
We already check for the minimum stake in the `removeUnapprovedNodes` method on line 837, so there is no need to check again here because any node with tokens in the committed bucket is already committed enough